dbfit-java/core/src/main/java/dbfit/fixture/Clean.java

```diff
@@ -81,10 +81,10 @@ public void deleteTableByIds(String tableName, String whereClause, String column
         }
 
         private void executeQuery(SQLQuery q) throws SQLException {
-            try (PreparedStatement st =
+            try (DbStatement st =
```

Comment:
Here it doesn't matter so much (it's 4 lines method) but I guess it makes sense to review all abbreviations (`cs`, `st`) and change them to `statement.